uvicorn/main.py

```diff
@@ -344,7 +342,7 @@ def print_version(ctx: click.Context, param: click.Parameter, value: bool) -> No
     "--h11-max-incomplete-event-size",
     "h11_max_incomplete_event_size",
     type=int,
-    default=DEFAULT_MAX_INCOMPLETE_EVENT_SIZE,
+    default=None,
```

Comment:
If I don't recall this wrong, I believe that when this was implemented `None` was the default, but it was changed to the default value that we currently have. I can be wrong tho...

Reply:
The value was None before [this PR](https://github.com/encode/uvicorn/pull/1534). IMO exposing the default value was just a side effect of fixing the breaking change in h11.